scripts/assign_workflow_user_library.py

```diff
@@ -4,27 +4,19 @@
 
 class AssignWorkflowUserPreparedLibrary(StepEPP):
     """
-    Assigns a plate created in User Prepared Library to either the Nano or PCR Free workflow.
+    Assigns a plate created in User Prepared Library to the Make and read qPCR Quant in the Nano workflow
+    - User Prepared Libraries are protocol "agnostic" so always go to Nano.
     """
+
     def _run(self):
-        artifact_to_route_pcr_free = set()
-        artifact_to_route_nano = set()
+        artifact_to_route = set()
 
         for art in self.output_artifacts:
-            sample = art.samples[0]
-            if sample.udf.get("Prep Workflow") == "TruSeq PCR-Free DNA Sample Prep":
-                artifact_to_route_pcr_free.add(art)
-
-            elif sample.udf.get("Prep Workflow") == "TruSeq Nano DNA Sample Prep":
-                artifact_to_route_nano.add(art)
-
-        if artifact_to_route_pcr_free:
-            stage = get_workflow_stage(self.lims, "TruSeq PCR-Free DNA Sample Prep", "SEMI-AUTOMATED - Make and Read qPCR Quant")
-            self.lims.route_artifacts(list(artifact_to_route_pcr_free), stage_uri=stage.uri)
+            artifact_to_route.add(art)
```

Comment:
This loop seems now unnecessary
You should route the output artifacts directly
```python
self.lims.route_artifacts(self.output_artifacts, stage_uri=stage.uri)
```

Reply:
Updated